packages/backend/local_database/seed.json

```diff
@@ -14,6 +14,13 @@
       }
     }
   },
+  "streams": [
+    {
+      "streamAddress": "0x86c6C2c9699bE74278E0d73065fF12249221Bd30",
+      "builderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+      "lastIndexedBlock": 0
+    }
+  ],
```

Comment:
I'm meeting with Austin tomorrow and we'll discuss what stream data we want to show in the UI. This will allow me to better understand what data structure we need.

This is not important for the dbLocal (only used for local dev), but it's for dbFirebase.

Some thoughts I have rn:
- I think that we want to store the stream data inside the users collection. Something like:

```json
    "0x70997970C51812dc3A010C7d01b50e0d17dc79C8": {
      "creationTimestamp": 1633088553223,
      "role": "builder",
      "stream": {
           "streamAddress": "0x86c6C2c9699bE74278E0d73065fF12249221Bd30",
            "lastIndexedBlock": 0,
            // Some extra data: Balance, stream size, etc
         }
      }
```

- We will use the `events` collection to store all the contract events. Something like:

```json
{
      "type": "stream.withdraw",
      "timestamp": 1645005681376,
      "payload": {
        "userAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
        "streamAddress": "0x86c6C2c9699bE74278E0d73065fF12249221Bd30",
        "amount": "0.5",
        // ....
      }
    }
```

I'll keep you posted!

Reply:
I've updated the function so it takes the stream info from the users collection, and stores events in the events collection.